src/Magento/FunctionalTestingFramework/Module/MagentoWebDriver.php

```diff
@@ -379,6 +379,48 @@ public function waitForPageLoad($timeout = null)
         $this->waitForLoadingMaskToDisappear($timeout);
     }
 
+    /**
```

Comment:
Please move this actions to separate Class which is going to be included only in pwa tests configuration https://github.com/magento-research/pwa-tests/blob/master/dev/tests/acceptance/tests/functional.suite.yml#L13

Reply:
Separate Class created.